kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/SectionSidePanel/SectionEditor.vue

```diff
@@ -458,10 +452,12 @@
     left: 0;
     display: flex;
     justify-content: space-between;
-    padding: 1em;
+    padding: 1em 2em;
     margin-top: 1em;
     background-color: #ffffff;
-    border-top: 1px solid black;
+    box-shadow:
+      0 0 2px rgba(0, 0, 0, 0.9),
```

Comment:
Just noting that we should probably be using our KDS dropdshadows https://design-system.learningequality.org/styling#elevation. (If this option was already considered and it was decided to burn these new values, then you can ignore my comment :sweat_smile:).